ssz-rs/examples/container_with_some_types.rs

```diff
@@ -53,15 +52,15 @@ fn main() {
     let encoding = match serialize(&foo) {
         Ok(encoding) => encoding,
         Err(e) => {
-            eprintln!("some error encoding: {}", e);
+            eprintln!("some error encoding: {:?}", e);
             return;
         }
     };
 
     let mut restored_foo = match Foo::<4>::deserialize(&encoding) {
         Ok(value) => value,
         Err(e) => {
-            eprintln!("some error decoding: {}", e);
+            eprintln!("some error decoding: {:?}", e);
```

Comment:
should be able to drop this change if we add a `fmt::Display` for `SimpleSerializeError`

Reply:
Addressed in https://github.com/ralexstokes/ssz-rs/pull/25/commits/b58448c30caa3cfc15bf696c020abb01001cac89.